### ISSUE TITLE
Empty ID and StreamID when simulcasting

### ISSUE THREAD

**Issue by jech** [pion/webrtc#1808]
Browser is offerer, and offers a PC with a single sendonly transceiver with two simulcast tracks.  When Pion fires OnTrack, both track.StreamID() and track.ID() are empty.

This does not happen if either of the following is true:
  * there is an audio track in the same PC in addition to the simulcast transceiver;
  * the client is not sending a simulcast track.

The offer sent by the browser is correct as far as I can see (it contains a well-formed a=msid line in the simulcast m section).

**Comment by zyxar** [comment]
I think I've seen similar cases (null ids) in our project. However we avoided using theses IDs directly in logic.

**Comment by Sean-Der** [comment]
Found the issue! Should have a fix soon.

libwebrtc is able to handle a `SetRemoteDescription` with one media section and no SSRC. We added similar code [here](https://github.com/pion/webrtc/blob/master/peerconnection.go#L1328-L1329)

So we have two things to fix here.
* We should populate StreamID/ID for non-SSRC single media sections
* We should assert that the MediaSection doesn't have a RID/Simulcast
